msl_expressions/autogenerated/src/Plans/Behaviours/Intercept.cpp

```diff
@@ -56,7 +57,7 @@ namespace alica
     void Intercept::run(void* msg)
     {
         /*PROTECTED REGION ID(run1458757170147) ENABLED START*/ //Add additional options here
-        this->setSuccess(true);
+        //this->setSuccess(true);
```

Comment:
This is probably from intercept refactor. This commit could probably removed.

Reply:
Commit cannot be removed, because there are some other changes in it.